tests/scalers_go/new_relic/new_relic_test.go

```diff
@@ -0,0 +1,270 @@
+//go:build e2e
+// +build e2e
+
+package new_relic_test
+
+import (
+	"encoding/base64"
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/joho/godotenv"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"k8s.io/client-go/kubernetes"
+
+	. "github.com/kedacore/keda/v2/tests/helper"
+)
+
+// Load environment variables from .env file
+var _ = godotenv.Load("../../.env")
+var now = time.Now()
+
+const (
+	testName = "new-relic-test"
+)
+
+var (
+	testNamespace           = fmt.Sprintf("%s-ns", testName)
+	deploymentName          = fmt.Sprintf("%s-deployment", testName)
+	serviceName             = fmt.Sprintf("%s-service-%v", testName, now.Unix())
```

Comment:
Is the random suffix any helpful since rest of the properties (testNamespace, deploymentName etc) remains the same.
I think that it is okay, to remove the random suffix.

Reply:
Shall I remove it? if you run this set of tests multiple times in the span of 90 seconds you might get flaky tests though because in New Relic, that's the same app.

Reply:
I'm not sure if we are going to trigger the e2e test that often, if it helps, we can use it. We are already using this approach of generating random numbers for [AWS scalers](https://github.com/kedacore/keda/blob/main/tests/scalers_go/aws_dynamodb/aws_dynamodb_test.go#L134), maybe you could just use the same helper